test/miner_poc_SUITE.erl

```diff
@@ -70,7 +70,7 @@ all() ->
      poc_dist_v8_test,
      poc_dist_v8_partitioned_test,
      poc_dist_v8_partitioned_lying_test,
-     poc_dist_v9_test,
+     poc_dist_v10_test,
```

Comment:
I seemed to have missed adding the other two v9 tests here, can you please add them for v10?

Reply:
while you're in here, can we comment out the v7 tests?